tracing-core/src/field.rs

```diff
@@ -748,7 +748,7 @@ impl<'a> ValueSet<'a> {
     /// Visits all the fields in this `ValueSet` with the provided [visitor].
     ///
     /// [visitor]: super::Visit
```

Comment:
making this public is breaking the docs build, because the link reference was actually always wrong, but RustDoc didn't notice because it wasn't in a public API: https://app.netlify.com/sites/tracing-rs/deploys/61ed8dcc6a225d0007090ba2

I think it should just be
```suggestion
    /// [visitor]: Visit
```
for this to build correctly?